app/common/containers/pages/ClinicDetailPage/index.js

```diff
@@ -32,17 +32,19 @@ import styles from './styles.scss';
 @withStyles(styles)
 @translate()
 @provideHooks({
-  fetch: ({ dispatch, params: { id } }) => Promise.all([
-    dispatch(fetchClinic(id)),
-  ]),
+  fetch: ({ dispatch, params: { id } }) =>
+    Promise.all([dispatch(fetchClinic(id))])
```

Comment:
Looks like there is no need in `Promise.all`

Reply:
yes)